File: common/src/main/java/com/wdz/common/net/response/NaviResponse.java
package com.wdz.common.net.response;

import java.util.List;

public class NaviResponse {
    public List<Data> data;

    @Override
    public String toString() {
        return "NaviResponse{" +
                "data=" + data +
                '}';
    }

    class Data{
        public List<Articles> articles;
        public int cid;
        public String name;

        @Override
        public String toString() {
            return "Data{" +
                    "articles=" + articles +
                    ", cid=" + cid +
                    ", name='" + name + '\'' +
                    '}';
        }

        class Articles{
            public String apkLink;
            public int audit;
            public String author;
            public boolean canEdit;
            public int chapterId;
            public String chapterName;
            public boolean collect;
            public int courseId;
            public String desc;
            public String descMd;
            public String envelopePic;
            public boolean fresh;
            public int id;
            public String link;
            public String niceDate;
            public String niceShareDate;
            public String origin;
            public String prefix;
            public String projectLink;
            public int publishTime;
            public int realSuperChapterId;
            public int selfVisible;
            public int shareDate;
            public String shareUser;
            public int superChapterId;
            public String superChapterName;
            public List<String> tags;
            public String title;
            public int type;
            public int userId;
            public int visible;
            public int zan;

            @Override
            public String toString() {
                return "Articles{" +
                        "apkLink='" + apkLink + '\'' +
                        ", audit=" + audit +
                        ", author='" + author + '\'' +
                        ", canEdit=" + canEdit +
                        ", chapterId=" + chapterId +
                        ", chapterName='" + chapterName + '\'' +
                        ", collect=" + collect +
                        ", courseId=" + courseId +
                        ", desc='" + desc + '\'' +
                        ", descMd='" + descMd + '\'' +
                        ", envelopePic='" + envelopePic + '\'' +
                        ", fresh=" + fresh +
                        ", id=" + id +
                        ", link='" + link + '\'' +
                        ", niceDate='" + niceDate + '\'' +
                        ", niceShareDate='" + niceShareDate + '\'' +
                        ", origin='" + origin + '\'' +
                        ", prefix='" + prefix + '\'' +
                        ", projectLink='" + projectLink + '\'' +
                        ", publishTime=" + publishTime +
                        ", realSuperChapterId=" + realSuperChapterId +
                        ", selfVisible=" + selfVisible +
                        ", shareDate=" + shareDate +
                        ", shareUser='" + shareUser + '\'' +
                        ", superChapterId=" + superChapterId +
                        ", superChapterName='" + superChapterName + '\'' +
                        ", tags=" + tags +
                        ", title='" + title + '\'' +
                        ", type=" + type +
                        ", userId=" + userId +
                        ", visible=" + visible +
                        ", zan=" + zan +
                        '}';
            }
        }
    }
}

File: kt_common/src/main/java/com/wdz/ktcommon/http/response/NaviResponse.kt
package com.wdz.ktcommon.http.response

class NaviResponse(var data: List<Data>) {


    data class Data(var articles: List<Articles>) {


        data class Articles(
            var apkLink: String,
            var audit: Int,
            var author: String,
            var canEdit: Boolean,
            var chapterId: Int,
            var chapterName: String,
            var collect: Boolean,
            var courseId: Int,
            var desc: String,
            var descMd: String,
            var envelopePic: String,
            var fresh: Boolean,
            var id: Int,
            var link: String,
            var niceDate: String,
            var niceShareDate: String,
            var origin: String,
            var prefix: String,
            var projectLink: String,
            var publishTime: Int,
            var realSuperChapterId: Int,
            var selfVisible: Int,
            var shareDate: Int,
            var shareUser: String,
            var superChapterId: Int,
            var superChapterName: String,
            var tags: List<String>,
            var title: String,
            var type: Int,
            var userId: Int,
            var visible: Int,
            var zan: Int = 0
        ) {


        }
    }
}